renderer/brave_content_settings_observer_browsertest.cc

```diff
@@ -499,8 +563,13 @@ IN_PROC_BROWSER_TEST_F(BraveContentSettingsObserverBrowserTest, AllowReferrer) {
 
   EXPECT_STREQ(ExecScriptGetStr(kReferrerScript,
       contents()).c_str(), "");
+  EXPECT_EQ(last_referrer(url()), "");
+  EXPECT_EQ(ExecScriptGetStr(create_image_script(), contents()),
```

Comment:
expected argument typically comes first, actual - second

Reply:
Thanks that's good to know.

I think it will leave them as they are right now because the whole file seems to be doing it wrong and fixing it everywhere would bloat my PR quite a bit.